src/Nri/Colors.elm

```diff
@@ -66,6 +102,56 @@ import Css exposing (hex, rgba)
 
 {-|
 
+<p style="font-size:2em; color: #00cbeb">#00cbeb</p>
+
```

Comment:
💭  In the new style guide, colors have usage descriptions, so I think we should add that in the docs:  For example, for `aqua`: "Mastery Level 2".

<img width="227" alt="screen shot 2017-07-20 at 6 19 14 pm" src="https://user-images.githubusercontent.com/1222/28445245-05b522c2-6d78-11e7-975d-c1a819507e68.png">